### PR TITLE
Migrated recent two blogs

### LINKED ISSUE #825: [NG] Migrate recent two blogs to Layer5-ng
**Current Behavior**
Recently, two [blogs](https://layer5.io/blog) have been added to layer5.

**Desired Situation**
Migrate those two blogs to the layer5-ng website.

- [ ] [Announcing Meshery v0.4.0](https://layer5.io/blog/meshery/announcing-meshery-v0.4.0)
- [ ] [My Journey from a contributor to maintainer](https://layer5.io/blog/community/my-journey-from-a-contributor-to-a-maintainer)

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by manishdangi98**
**Description**

This PR fixes #825

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@manishdangi98! 👍

File: src/collections/blog/2020-08-11-my-journey-from-a-contributor-to-a-maintainer/index.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Let’s begin with the basics. What is Layer5?
+
+Layer5 is the service mesh company, which has built (or is in the process of building) an exhaustive set of projects to delve into the world of cloud native and service meshes with in-depth analyses. Intrigued? Checkout the [Layer5](https://layer5.io/) site and hop on to our [GitHub](https://github.com/layer5io) bandwagon. Before you do me the favor of reading on, make sure to click on the little star :star: button to help us stir up some more delectable meshiness.
+
+#### Roadblocks
+
+Coming to my contributions at Layer5, I have mostly been involved in improving the UI and shaping the company site’s present outlook. I have also tried my hand at other projects, including the `mesheryctl` and the `landscape` at Layer5. Here is...
+
+<div class="center" style="padding:5px;background:#eee;"> <h4>a comprehensive list of <a href="https://github.com/issues?page=3&q=is%3Aclosed+author%3ANikhil-Ladha+org%3Alayer5io">my contributions across Layer5 projects</a></h4></div>
```
Comment (Jashpatel1) [inline src/collections/blog/2020-08-11-my-journey-from-a-contributor-to-a-maintainer/index.mdx:40]: This styles can be omitted.

File: src/collections/blog/2020-08-11-my-journey-from-a-contributor-to-a-maintainer/index.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+import { BlogWrapper } from "../Blog.style.js";
+import pic1 from "./pic1.png";
+import pic2 from "./pic2.jpeg";
+
+<BlogWrapper>
+
+Getting involved in something new and essentially avant garde, can be a messy challenge to navigate through. But, when it comes to contributing to Meshery it only gets more and more meshy!
+<img src={pic1} class="image-right"/>
+#### Get set go!
```
Comment (Jashpatel1) [inline src/collections/blog/2020-08-11-my-journey-from-a-contributor-to-a-maintainer/index.mdx:22]: This is rendered as it is and not as a heading. Please retry doing this.

Comment (manishdangi98) [reply src/collections/blog/2020-08-11-my-journey-from-a-contributor-to-a-maintainer/index.mdx:22]: Do I need to remove this? As the same image is repeated here.

Comment (Tanuj22) [reply src/collections/blog/2020-08-11-my-journey-from-a-contributor-to-a-maintainer/index.mdx:22]: Yes. Remove the image. Also leaving a line between the heading and the paragraph will fix the rendering as it issue.

File: src/collections/blog/2020-08-07-announcing-meshery-v0.4/index.mdx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+**Exposing Performance Management in the CLI**<br/>
+`perf:` a new `mesheryctl`command. Introduction of new performance sub-commands, now benchmark your service mesh at the tip of your fingers using our new CLI command `perf`.
+
+**Support for Scoop** <br/>
+Support extended to Scoop Bucket. You can now install mesheryctl on your Windows machine with Scoop Bucket. Visit the [Meshery Scoop Bucket](https://github.com/layer5io/scoop-bucket) to install Meshery on Windows.
+
+**Rename `cleanup` to `reset`**<br/>
+The `cleanup` is used to reset your Meshery deployment configuration back to its default settings. This command has been renamed to `reset` to more appropriately reflect its purpose.
+
+#### MeshSync<br/>
```
Comment (Tanuj22) [inline src/collections/blog/2020-08-07-announcing-meshery-v0.4/index.mdx:48]: Increase size

File: src/collections/blog/2020-08-07-announcing-meshery-v0.4/index.mdx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+<img src={operator} class="image-right"/>
+
+-  A component of the [Meshery Operator](https://github.com/layer5io/meshery-operator), MeshSync can scan the environment to get the deployment details of specific types of service meshes and the connected Kubernetes cluster.
+-  MeshSync is a new component addition to Meshery. Meshery needs to be constantly updated given that service meshes and their underlying infrastructure are dynamic, constantly changing.  Meshery operations should be resilient in the face of this change.
+-  MeshSync brings a service mesh agnostic object model that defines relationships between all objects under management.
+
+#### Meshery Adapter for Citrix Service Mesh (beta)<br/>
```
Comment (Tanuj22) [inline src/collections/blog/2020-08-07-announcing-meshery-v0.4/index.mdx:55]: Increase size

File: src/collections/blog/2020-08-07-announcing-meshery-v0.4/index.mdx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+<img src={architecture} class="image-right"/>
+
+- [Citrix Service Mesh](https://github.com/layer5io/meshery-cpx) is now a supported service mesh. Meshery incorporates support for the Citrix ADC CPX, which is a cloud-ready, container-based application delivery controller that can be provisioned on a Docker host.
+- CPX runs as the Istio Data Plane component, displacing Envoy as the default data plane service proxy.<br/>
+
+#### Security & Authentication
```
Comment (Tanuj22) [inline src/collections/blog/2020-08-07-announcing-meshery-v0.4/index.mdx:62]: Increase size

File: src/collections/blog/2020-08-07-announcing-meshery-v0.4/index.mdx [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+- Meshery has moved from using session authentication to JWT authentication. Meshery's JWT authentication is powered by Hydra Auth.
+- You can now opt to authenticate yourself on mesheryctl while performing performance tests using `mesheryctl`, you can authenticate yourself by getting the JWT Token from Meshery UI.
+
+#### Meshery Server
```
Comment (Tanuj22) [inline src/collections/blog/2020-08-07-announcing-meshery-v0.4/index.mdx:67]: Increase size

File: src/collections/blog/2020-08-07-announcing-meshery-v0.4/index.mdx [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+- Support provided for [wrk2 as an alternative load generator](https://meshery.layer5.io/docs/functionality/performance-management#load-generators).
+- [Providers](https://meshery.layer5.io/docs/extensibility) - A new project construct that allows users to select authentication, long-term storage etc.
+- Ad-hoc [connectivity tests for Prometheus and Grafana](https://meshery.layer5.io/docs/functionality/performance-management#grafana-and-meshery) are now supported.
+- Extraneous information beyond IP address and port in Grafana and Prometheus endpoints have been stripped off.
+
+#### Meshery UI
```
Comment (Tanuj22) [inline src/collections/blog/2020-08-07-announcing-meshery-v0.4/index.mdx:74]: Increase size

File: src/collections/blog/2020-08-07-announcing-meshery-v0.4/index.mdx [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+
+- ES-Lint has been added to the client side to ensure the quality of code and increase maintainaiblity of code.
+- Cypress has been set-up to enable end-to-end tests and integration tests for Meshery UI.
+
+<center><iframe frameborder="0" width="100%" height="300" allowfullscreen={true} mozallowfullscreen="true" webkitallowfullscreen="true"frameborder="0" width="100%" height="300" allowfullscreen={true} mozallowfullscreen="true" webkitallowfullscreen="true" src="https://www.youtube.com/embed/ds9D2KgZKxo" ></iframe></center>
+
+
+#### Other notable changes
```
Comment (Tanuj22) [inline src/collections/blog/2020-08-07-announcing-meshery-v0.4/index.mdx:82]: Increase size

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
`a comprehensive list of ....` should be something like the current site 

Screenshot from 2020-10-12 22-34-25

**Review by Jashpatel1 — APPROVED** [review]
Thanks @manishdangi98. I am a bit concerned about the update of gatsby-plugin @Tanuj22 . Rest looks fine.

**Review by Tanuj22 — APPROVED** [review]
I see no problem with updating the plugin. Thank for working fon this @manishdangi98

**Pull request merged**